app/views/api/v2/analytics/user_analytics/_monthly_gender_table.html.erb

```diff
@@ -0,0 +1,48 @@
+<!-- the "all" table -->
+<table class="progress-table <%= data_table_name %> hypertension:all">
+  <tr class="total">
+    <td class="row-label">All-time</td>
+    <td>
+      <%= zero_if_unavailable(@statistics
+                                  .dig(:all_time, :grouped_by_gender, resource)
+                                  .values
+                                  .inject(:+)) %>
+    </td>
+  </tr>
+  <% @monthly_period_list.each do |month_date| %>
+    <tr>
+      <td class="row-label no-padding"><%= month_date.strftime("%B") %></td>
+      <td>
+        <%= zero_if_unavailable(@statistics
+                                    .dig(:monthly, :grouped_by_gender_and_date, resource)
+                                    .values
+                                    .inject { |acc, next_item| acc.merge(next_item) { |_, v1, v2| v1 + v2 } }
+                                    &.dig(month_date)) %>
```

Comment:
There's a lot going on here. How can we simplify this for fellow devs?

Reply:
Addressed in https://github.com/simpledotorg/simple-server/pull/843/commits/df92264398c30997e9fe4571b9ac938becce08ff.